packages/gvim.rb

```diff
@@ -3,81 +3,55 @@
-            --enable-gui \
-            --with-x \
-            --enable-fontset \
-            --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --enable-python3interp \
-            --enable-rubyinterp \
-            --with-python3-command=#{CREW_PREFIX}/bin/python3 \
-            --with-ruby-command=#{CREW_PREFIX}/bin/ruby \
+            --localstatedir=#{CREW_PREFIX}/var/lib/vim \
             --with-features=huge \
+            --with-compiledby='Chromebrew' \
+            --with-x=yes \
+            --enable-gui=gtk2 \
+            --enable-multibyte \
+            --enable-cscope \
+            --enable-perlinterp=dynamic \
+            --enable-pythoninterp=dynamic \
+            --enable-python3interp=dynamic \
+            --enable-rubyinterp=dynamic \
             --disable-selinux"
-    system 'make'
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81"
   end
 
   def self.install
-    tmpdir = "#{CREW_PREFIX}/tmp/vim"
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    configdestdir = "#{CREW_DEST_PREFIX}/.config/vim"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    FileUtils.mkdir_p "#{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/evi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/*gvi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/evi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/*gvi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/vim/vim81/gvimrc_example.vim #{tmpdir}"
-    system "rm -rf #{CREW_DEST_PREFIX}"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/gvimrc_example.vim #{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/*.1 #{CREW_DEST_PREFIX}/share/man/man1"
-    system "mv #{tmpdir}/* #{CREW_DEST_PREFIX}/bin"
-    system "rm -rf #{tmpdir}"
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{configdestdir}"
-    FileUtils.mkdir_p "#{CREW_DEST_HOME}"
-    system "touch #{configdir}/gvimrc #{configdestdir}/gvimrc"
-    system "ln -s #{configdir}/gvimrc #{CREW_DEST_HOME}/.gvimrc"
-  end
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81", "DESTDIR=#{CREW_DEST_DIR}", "install"
 
-  def self.postinstall
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    system "cp #{CREW_PREFIX}/share/vim/vim81/gvimrc_example.vim #{configdir}/gvimrc" unless File.size? "#{configdir}/gvimrc"
-    puts
-    puts "To finish the installation, execute the following:".lightblue
-    puts "echo '# Fixes Warning: Missing charsets in String to FontSet conversion' >> ~/.bashrc".lightblue
-    puts "echo 'export LC_ALL=C' >> ~/.bashrc && . ~/.bashrc".lightblue
-    puts
-    puts "The config files are located in $HOME/.vim".lightblue
```

Comment:
Modify this and the corresponding line in `vim.rb` to say `    puts "The config files are located in #{CREW_PREFIX}/etc".lightblue`

Reply:
Yes, I have to bring the instructions back, especially to warn about the `export VIMRUNTIME` that will break things.